src/seedu/addressbook/commands/DeleteMCommand.java

```diff
@@ -0,0 +1,53 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Deletes a person identified using it's last displayed index from the address book.
+ */
+public class DeleteMCommand extends Command{
+    public static final String COMMAND_WORD = "deleteM";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes people identified by their comma seperated index numbers used in the last find/list call.\n"
+            + "Parameters: COMMER SEPERATED INDEXES\n"
+            + "Example: " + COMMAND_WORD + " 1,2,3";
+
+    //TODO:
```

Comment:
Please provide more details for TODO

Reply:
Sure. I was thinking to let it support more formats.